frontend/src/components/RestaurantCard.tsx

```diff
@@ -40,7 +41,7 @@ export default function RestaurantCard({ onAcceptButtonClick, onDeclineButtonCli
                   <p>{cuisine}</p>
                   <p>{rating}</p>
                   <div className="h-48 w-96">
-                      <img className="object-cover h-48 w-72" src={image_url} />
+                      <Image className="object-cover h-48 w-72" src={image_url} alt="hi" width={50} height={50} />
```

Comment:
using next's image component here. we'd probably want to modify the width and height, but those (along with `alt`) are required attributes so i just threw in dummy data for now